app/src/main/res/layout/fragment_new_message.xml

```diff
@@ -284,34 +284,38 @@
                         tools:listitem="@layout/item_attachment"
                         tools:visibility="visible" />
 
-                    <com.google.android.material.textfield.TextInputLayout
-                        android:id="@+id/bodyLayout"
+                    <FrameLayout
+                        android:id="@+id/editorLayout"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        app:boxBackgroundColor="@android:color/transparent"
-                        app:boxBackgroundMode="none"
-                        app:boxStrokeWidth="0dp"
-                        app:hintEnabled="false"
+                        android:paddingTop="@dimen/marginStandardMedium"
                         app:layout_constraintEnd_toEndOf="parent"
                         app:layout_constraintStart_toStartOf="parent"
-                        app:layout_constraintTop_toBottomOf="@id/attachmentsRecyclerView"
-                        app:placeholderText="@string/newMessagePlaceholderTitle"
-                        app:placeholderTextColor="@color/tertiaryTextColor">
+                        app:layout_constraintTop_toBottomOf="@id/attachmentsRecyclerView">
 
-                        <com.google.android.material.textfield.TextInputEditText
-                            android:id="@+id/bodyTextField"
-                            style="@style/Body"
+                        <com.infomaniak.lib.richhtmleditor.RichHtmlEditorWebView
+                            android:id="@+id/editor"
                             android:layout_width="match_parent"
-                            android:layout_height="wrap_content"
-                            android:gravity="top"
-                            android:hint="@string/newMessagePlaceholderTitle"
-                            android:inputType="textMultiLine|textCapSentences"
-                            android:nextFocusUp="@id/subjectTextField"
-                            android:paddingBottom="0dp"
-                            android:visibility="gone"
-                            tools:visibility="visible" />
+                            android:layout_height="wrap_content" />
```

Comment:
Do we need to have 2 levels here (FrameLayout -> Editor) instead of just one?

Reply:
I thought it was needed for something but I must have been thinking of something else because it doesn't look it's needed here